satpy/etc/readers/viirs_edr_active_fires.yaml

```diff
@@ -5,44 +5,64 @@ reader:
     sensors: [viirs]
 
 file_types:
+    fires_netcdf_img:
+        file_reader: !!python/name:satpy.readers.viirs_edr_active_fires.VIIRSActiveFiresFileHandler
+        variable_prefix: ""
+        file_patterns:
+            - 'AFIMG_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.nc'
     fires_netcdf:
         file_reader: !!python/name:satpy.readers.viirs_edr_active_fires.VIIRSActiveFiresFileHandler
+        variable_prefix: "Fire Pixels/"
         file_patterns:
-            - 'AFEDR_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.nc'
+        - 'AFEDR_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.nc'
+        - 'AFMOD_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.nc'
     fires_text:
         file_reader: !!python/name:satpy.readers.viirs_edr_active_fires.VIIRSActiveFiresTextFileHandler
         file_patterns:
-            - 'AFEDR_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.txt'
+            - '{satellite_name}_npp_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time}_{source}.txt'
 
 datasets:
-    detection_confidence:
-        name: detection_confidence
-        standard_name: fire_confidence
+    confidence_cat:
+        name: confidence_cat
+        standard_name: confidence_cat
+        file_type: [fires_text, fires_netcdf_img]
+        file_key: "{variable_prefix}FP_confidence"
+        coordinates: [longitude, latitude]
+        units: '[7,8,9]->[lo,med,hi]'
+    confidence_pct:
+        name: confidence_pct
+        standard_name: confidence_pct
         file_type: [fires_netcdf, fires_text]
-        file_key: "Fire Pixels/FP_confidence"
+        file_key: "{variable_prefix}FP_confidence"
         coordinates: [longitude, latitude]
         units: '%'
     longitude:
         name: longitude
         standard_name: longitude
-        file_type: [fires_netcdf, fires_text]
-        file_key: "Fire Pixels/FP_longitude"
+        file_type: [fires_netcdf, fires_text, fires_netcdf_img]
```

Comment:
Same for these, netcdf for IMG should be first, then netcdf for MOD, then text for img, then text for mod.